### ISSUE TITLE
Getting process infos results in exception

### ISSUE THREAD

**Issue by mattkol** [MicrosoftEdge/WebView2Feedback#2589]
**Description**
Using Blazor MAUI, I am unable to get process infos. Results in exception when trying to execute WebViewEnvironment.GetProcessInfos();

**Version**
SDK:  n/a
Runtime: 103.0.1264.49
Framework: Blazor MAUI
OS: Win11

**Repro Steps**
Create a simple default Blazor MAUI project and make following changes and run:

MainPage.xaml:
````c#
<BlazorWebView HostPage="wwwroot/index.html" x:Name="_blazorWebView">
      <BlazorWebView.RootComponents>
          <RootComponent Selector="#app" ComponentType="{x:Type local:Main}" />
      </BlazorWebView.RootComponents>
</BlazorWebView>
````
MainPage.xaml.cs:
````c#
public partial class MainPage : ContentPage
{
	public MainPage()
	{
	    InitializeComponent();
            _blazorWebView.BlazorWebViewInitialized += OnBlazorWebViewInitialized;
         }

    public uint BrowserProcessId { get; internal set; }
    public string RuntimeVersion { get; internal set; }

#if WINDOWS
        CoreWebView2Environment WebViewEnvironment { get; set; }
#endif

    private void OnBlazorWebViewInitializing(object sender, Microsoft.AspNetCore.Components.WebView.BlazorWebViewInitializingEventArgs e)
    {
    }

    private void OnBlazorWebViewInitialized(object sender, Microsoft.AspNetCore.Components.WebView.BlazorWebViewInitializedEventArgs e)
    {
#if WINDOWS
        BrowserProcessId = e.WebView.CoreWebView2.BrowserProcessId;
        WebViewEnvironment = e.WebView.CoreWebView2.Environment;
        RuntimeVersion = WebViewEnvironment.BrowserVersionString;
        WebViewEnvironment.ProcessInfosChanged += WebView_ProcessInfosChanged;
#endif
    }

#if WINDOWS
    void WebView_ProcessInfosChanged(object sender, object e)
    {
       // *** Exception occurs here
        var processInfos = WebViewEnvironment.GetProcessInfos();
    }
#endif

}
````
[ProcessInfosMAUI.zip](https://github.com/MicrosoftEdge/WebView2Feedback/files/9106862/ProcessInfosMAUI.zip)

**Screenshots**

![get-process-infos-issue](https://user-images.githubusercontent.com/18384207/178858536-2a669b0e-2b02-4a42-b6ff-35ae2f82cf71.gif)

[AB#40543811](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/40543811)

**Comment by champnic** [comment]
This issue has been fixed and will be available for testing in the next prerelease SDK.

**Comment by champnic** [comment]
This is fixed in SDK 1.0.1481+
